### ISSUE TITLE
Parsing error when using extrapolated variable as extend name

### ISSUE THREAD

**Issue by micahgodbolt** [sasstools/sass-lint#313]
Getting parsing errors when I try to use an extrapolated variable in a silent extend.

```bash
Warning: Parsing error at test/fixtures/test1.scss: Please check validity of the block starting from line #3 Use --force to continue.
```

```sass
$var: 'foo';

%#{$var} {
  font-size: 18px;
}
```

**Comment by bgriffith** [comment]
Hey @micahgodbolt - thanks for the issue. It's currently being caused by the parser so I've added an issue. Here's a reference https://github.com/tonyganch/gonzales-pe/issues/110

**Comment by DanPurdy** [comment]
Confirmed as working in gonzales-pe 3.2.1

**Comment by danez** [comment]
As far as I can see gonzales-pe >= 3.0.0 fixed the issue. When I look at the PR it seems you are preparing to upgrade to version 3.2 soon, right? Do you guys have any eta when this will be finished? Or maybe upgrade to 3.0.x before 3.2 if there are no so many breaking changes in it?

We would love to use sass-lint, but this bug keeps us from doing that.

**Comment by DanPurdy** [comment]
3.0 contains as many breaking changes as 3.2.. there are still regressions in 3.2 that are going to need to be fixed before this is complete but fingers crossed! As for an eta, whenever we get the time to work on it, i'm afraid i can't be any more specific than that. 

Might be worth keeping an eye on this issue though, https://github.com/sasstools/sass-lint/issues/495

**Comment by bgriffith** [comment]
@micahgodbolt The parse error has been fixed in 1.6.

**Comment by DanPurdy** [comment]
ping @danez too ^^
